systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -118,6 +119,7 @@ function checkDirty(obj)
 
     function model = compile(model)
       w = warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
+      model.gurobi_present = checkDependency('gurobi');
```

Comment:
Also worth noting is that inside the checkDependency implementation, commenting out the try/catch block that actually calls gurobi to check for a license stopped the segfault.